test/evergreen.yml

```diff
@@ -1784,6 +1814,48 @@ tasks:
 
             ./time_shift_test.sh /usr/local/lib/faketime/libfaketimeMT.so.1 0-1 2>&1
 
+  - name: recovery-stress-test
+    #set a 25 hours timeout
+    exec_timeout_secs: 90000
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags: --enable-strict --enable-diagnostic --with-builtins=lz4,snappy,zlib
+      - func: "recovery stress test script"
+        vars:
+          # Repeat this script enough times to make this task run for 24 hours
+          # At the time of writing this script, one cal to underlying scripts takes about 8 mins to finish.
+          times: 180
```

Comment:
Should we leave some space in case some runs take longer than expected?

Reply:
I am running a full test that I am expecting to take around <= 24 hours. I will adjust this number once that job finished to make room for any variance in execution time.